buildifier/utils/utils.go

```diff
@@ -68,3 +72,47 @@ func GetParser(inputType string) func(filename string, data []byte) (*build.File
 		return build.ParseDefault
 	}
 }
+
+// WriteTemp writes data to a temporary file and returns the name of the file.
+func WriteTemp(data []byte, toRemove *[]string) (file string, err error) {
```

Comment:
C++ style "out" arguments like toRemove* are strongly discouraged.  

Why not add an extra return value?

Reply:
Removed the argument, delegated the logic to the caller of `WriteTemp`.